src/main/java/frc/robot/RobotContainer.java

```diff
@@ -57,15 +63,38 @@ private void configureButtonBindings() {
         .whenInactive(new StopOuttakeCommand(outtake));
   }
 
+  // A simple auto routine.
+  private final Command simpleAuto =
+      new SimpleAutonCommand(driveBase, Constants.SIMPLE_AUTON_SPEED,
+          Constants.SIMPLE_AUTON_RUNTIME);
+
+  // A medium auto routine.
+  private final Command mediumAuto =
+      new ParallelDeadlineGroup(new WaitCommand(Constants.MEDIUM_AUTON_OUTAKE_RUNTIME),
+          new RunOuttakeCommand(outtake, Constants.OuttakeConstants.OUTTAKE_SPEED),
+          new SimpleAutonCommand(driveBase, Constants.SIMPLE_AUTON_SPEED,
```

Comment:
Can be replaced with just "simpleAuto" (because we have already declared and initialized the simple auton with the exact same parameters)